### ISSUE TITLE
do not mark issues that are being cross referenced 

### ISSUE THREAD

**Issue by maxandersen** [probot/stale#255]
could it be added some flag to have stale-bot avoid closing issues that have cross-references ? 

similar issues: https://github.com/probot/stale/issues/154 and https://github.com/probot/stale/issues/73 ?

**Comment by Qix-** [comment]
libuv/libuv#2255 is an example of this.

Stalebot is crapping up issues that have progress elsewhere; it's getting _incredibly_ annoying.

Please, _please_ stop the bot commenting on issues that have active PRs, for example. This is not the first time I've had to battle with @stalebot for this same reason. It doesn't help anyone.

**Comment by Qix-** [comment]
![image](https://user-images.githubusercontent.com/885648/75727120-7563e000-5ce4-11ea-8ec7-4f30b7efeb79.png)

So much damn noise.

**Comment by maxandersen** [comment]
@Qix- I agree it should be possible to at least as an option implement such check but in the short run just add a "pinned" label on it and you avoid the frustration :)

**Comment by Qix-** [comment]
I'm not a maintainer of libuv so I cant to this.

**Comment by maxandersen** [comment]
stalebot marking its own issue stale is funny :)

Yes, its still relevant.

**Comment by Qix-** [comment]
Your own bot on your own issue tracker is proving the point of this issue...

I have come to _loathe_ this bot and generally steer clear of projects that use it now because it's more of a reflection of the maintainers and less something "useful" or impactful.

Perhaps I'm just grumpy, but this thing generates an insane amount of noise across Github.

**Comment by Qix-** [comment]
...

**Comment by Qix-** [comment]
So it asks if there's anything to do here since the maintainers don't respond whilst simultaneously marking it as "won't fix"?

**Comment by Qix-** [comment]
.................................................................................................................................................
